### PR TITLE
Installs functional test deps if pytest_args is specified.

### PR DESCRIPTION
**Pull request by dkliban**
Also renames TELEMETRY setting to ANALYTICS.

[noissue]

### CODE REVIEW

File: templates/github/.ci/ansible/settings.py.j2.copy [hunk 1]
```diff
@@ -8,7 +8,6 @@ TOKEN_SIGNATURE_ALGORITHM = "ES256"
 CACHE_ENABLED = True
 REDIS_HOST = "localhost"
 REDIS_PORT = 6379
-TELEMETRY = False
```
Comment (mdellweg) [inline templates/github/.ci/ansible/settings.py.j2.copy:11]: Don't remove that, replace it by ANALYTICS please.

**Review by mdellweg — CHANGES_REQUESTED** [review]

File: templates/github/.ci/ansible/settings.py.j2.copy (hunk 1 above)
Comment (dkliban) [reply templates/github/.ci/ansible/settings.py.j2.copy:11]: will do.

File: templates/github/.ci/ansible/settings.py.j2.copy [hunk 2]
```diff
@@ -8,7 +8,7 @@ TOKEN_SIGNATURE_ALGORITHM = "ES256"
...
+ANALYTICS = False
```
Comment (mdellweg) [inline templates/github/.ci/ansible/settings.py.j2.copy:11]: Thanks. The reason is that the dev.analytics... will still happily sum up all systems, as we set the `..._MIN_AGE=0` there.

**Review by mdellweg — APPROVED** [review]

**Comment by mdellweg** [comment]
I have no idea what the CI is doing on a monday again.

**Pull request merged**